### PR TITLE
AGDIGGER-2 - automated tests in travis

### PR DESCRIPTION
**Pull request by odra**
Done for:
https://issues.jboss.org/browse/AGDIGGER-2

Added some scripts to install android sdk and cordova do run tests with `py.test` in travis

Do not merge it yet :)

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,9 +1,22 @@
...
+sudo: false
+
+addons:
+  apt:
+    packages':
+    - expect
+
+script:
+  - "./tests/scripts/install-android-sdk"
+  - "./tests/scripts/install-android-packages"
```
Comment (matzew) [inline .travis.yml]: let's move them somewhere else 

.e.g. a `travis` folder ?

Comment (matzew) [reply .travis.yml]: @odra can't we move them to a `travis` folder ? 

they are really just for TravisCI, right ?

**Comment by matzew** [comment]
@cotsog any idea waht's going on here ? 

we are getting some

```
Cannot run program "/opt/android-sdk-linux/build-tools/23.0.2/aapt": error=2, No such file or directory` errors
```

when manually running `which /opt/android-sdk-linux/build-tools/23.0.2/aapt` it works

**Comment by cotsog** [comment]
@matzew: that's interesting. Could it be a credential issue? What's the user running the (Java?) program that's calling `/opt/android-sdk-linux/build-tools/23.0.2/aapt`? Can you show me the code responsible for calling it?

So that your issue isn't lost in the cracks, could you send an e-mail to support [at] travis-ci [dot] com?

Thanks!

**Comment by odra** [comment]
@cotsog besides installing nodejs/npm with `sudo` we are not using/creating any additional users.

The command `/opt/android-sdk-linux/build-tools/23.0.2/aapt` is invoked by `gradlew` which is invoked by our [tests](https://github.com/aerogear/digger-build-cli/tree/master/tests).

Also, we are using `sudo: required`.

**Comment by aliok** [comment]
sorry I saw this PR very late. this problem happened to me on my linux box. The binary `aapt` was there, but when I tried to run it, I was seeing the same error.
It turns out it is just missing dependencies on the OS level.

http://stackoverflow.com/questions/22701405/aapt-ioexception-error-2-no-such-file-or-directory-why-cant-i-build-my-grad

`apt-get install lib32stdc++6 lib32z1` should fix the issue.

**Pull request merged**